client/src/www/model/events.ts

```diff
@@ -36,15 +36,19 @@ export class ServerRenamed implements OutlineEvent {
 }
 
 export class ServerConnected implements OutlineEvent {
-  constructor(readonly server: Server) {}
+  constructor(readonly serverId: string) {}
+}
+
+export class ServerDisconnecting implements OutlineEvent {
```

Comment:
It turns out disconnecting events were not being propagated. The UI would triggeer the state on the start of the disconnect server action, but it wasn't triggered by an event. This is now fixed.